dask_ml/model_selection/_incremental.py

```diff
@@ -111,15 +118,22 @@ def _create_model(model, ident, **params):
 def _fit(
     model,
     params,
-    X_train,
-    y_train,
-    X_test,
-    y_test,
+    X_train: da.Array,
+    y_train: da.Array,
+    X_test: da.Array,
+    y_test: da.Array,
```

Comment:
Let's hold of on adding these types, I don't want it holding up this PR. I think the test versions at least can be numpy arrays too.